R/pkg/R/DataFrame.R

```diff
@@ -945,6 +947,19 @@ setMethod("[", signature(x = "DataFrame", i = "missing"),
             select(x, j)
           })
 
+#' @rdname select
+setMethod("[", signature(x = "DataFrame", i = "Column"),
+          function(x, i, j, ...) {
```

Comment:
So we need the `...` to match the signature of `[` in base R. From the help page you can see that the signatures look like

```
     x[i]
     x[i, j, ... , drop = TRUE]
     x[[i, exact = TRUE]]
     x[[i, j, ..., exact = TRUE]]
```